### PR TITLE
Add semi structured models for 22.4

### PR DESCRIPTION
**Pull request by irinel-birsan-uipath**

### CODE REVIEW

File: metadata/4506T__1__metadata.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "inputType": "JSON",
+    "displayName": "4506T",
+    "memory": 0,
+    "mlPackageLanguage": "PYTHON37_DU",
+    "projectId": "[project-id]",
+    "stagingUri": "[staging-uri]",
+    "projectName": "UiPath Document Understanding",
+    "projectDescription": "UiPath models to classify and extract information from images and pdfs.",
+    "tenantName": "UiPath",
+    "minAIFabricVersion": "v21.10",
```
Comment (irinel-birsan-uipath) [inline metadata/4506T__1__metadata.json:20]: @vivekjhauipath @vnaren23 Which is the min ai fabric version that work with language version 4?

Comment (vnaren23) [reply metadata/4506T__1__metadata.json:20]: Language version 4 will only go with AIC 22.4. Also version will be `22.4` and not ~~`v22.4`~~

Comment (irinel-birsan-uipath) [reply metadata/4506T__1__metadata.json]: Done

**Review by mohitseth-nitrr — APPROVED** [review]
LGTM

**Review by vnaren23 — APPROVED** [review]
LGTM

**Pull request merged**